### DISCUSSION TITLE
[version 2] Unrecognized files

### DISCUSSION THREAD

**Discussion by raysam** [a1ex4/ownfoil#55] [General]
I installed successfully version 2 on Synology Nas

I tried copy new release game (NSP type) to the games folder but it not show on the list or tinfoil new game. old game still work perfect can you help me or give me some guide 🫡 . Thanks

**Comment by JCStrainer** [comment]
I have the same issue, .nsz are found, .xcz, .xci and .nsp files not.

Is there a special naming convention or certain characters not supported?

**Comment by a1ex4** [comment]
There was an issue where only .nsp and .nsp files were filtered, I fixed it to also include .xci and .xcz, you can try by pulling the new v2 image.

Regarding the naming currently the files must respect 'Game [TITLEID][vVERSION]', only the brackets are important. Later I will add support to also recognize files that do not respect this convention.

Pease let me know if you can now add all your files, or put the filenames here if they are not recognized.

**Comment by raysam** [comment]
I pulled and build new Container and got this error :

2023-10-10 18:30:04 Starting ownfoil
2023-10-10 18:30:04 Traceback (most recent call last):
2023-10-10 18:30:04   File "/app/app.py", line 2, in <module>
2023-10-10 18:30:04     from flask_login import LoginManager
2023-10-10 18:30:04   File "/usr/local/lib/python3.11/site-packages/flask_login/__init__.py", line 12, in <module>
2023-10-10 18:30:04     from .login_manager import LoginManager
2023-10-10 18:30:04   File "/usr/local/lib/python3.11/site-packages/flask_login/login_manager.py", line 33, in <module>
2023-10-10 18:30:04     from .utils import _create_identifier
2023-10-10 18:30:04   File "/usr/local/lib/python3.11/site-packages/flask_login/utils.py", line 14, in <module>
2023-10-10 18:30:04     from werkzeug.urls import url_decode
2023-10-10 18:30:04 ImportError: cannot import name 'url_decode' from 'werkzeug.urls' (/usr/local/lib/python3.11/site-packages/werkzeug/urls.py)

**Comment by a1ex4** [comment]
@raysam Should be fixed now, it's because of the Flask/Werkzeug update to 3.0.0 and Flask-Login not being compatible yet.

**Comment by fettesb** [comment]
World of Outlaws Dirt Racing '23 Edition [01006E801B044000][v65536][US].nsp will not show up. I re-pulled the new container

**Comment by a1ex4** [comment]
@fettesb this is a bug in the cnmts.json from Tinfoil database, this specific game is missing. I pushed a commit that contains a workaround if a base title is missing from cnmts, it should fix your problem.

**Comment by acaltas** [comment]
is the issue back.. with the new container image only cxi files are shown in the GUI rest is skipped

**Comment by a1ex4** [comment]
@acaltas please provide examples of unrecognized files

**Comment by acaltas** [comment]
example
9.Monkeys.Of.Shaolin [010005E00E2BC000].nsp is not shown

Animal.Crossing.New.Horizons [01006F8002326000].xci is shown

but King_of_seas[01008D80148C8000].nsp is working again

**Comment by SiscuPrats** [comment]
In my case... .nsz is shown, but generating list is very slow.

New file to add: /games/Radio Commander [0100BAD013B6E000][US][v0].nsz                                                                                                                                                                       
New file to add: /games/Radio Squid [0100655012064000][US][v0].nsz                                                                                                                                                                           
New file to add: /games/Radioactive Dwarfs Evil From the Sewers [0100DCF017B4A000][US][v0].nsz                                                                                                                                               
New file to add: /games/RADIOHAMMER STATION [01008FA00ACEC000][US][v0].nsz                                                                                                                                                                   
New file to add: /games/RADIOHAMMER STATION [01008FA00ACEC800][US][v65536].nsz                                                                                                                                                               
New file to add: /games/Radirgy Swag [01000B900EEF4000][US][v0].nsz                                                                                                                                                                          
New file to add: /games/Radon Blast [0100323014580000][US][v0].nsz                                                                                                                                                                           
New file to add: /games/Radon Blast [0100323014580800][US][v65536].nsz                                                                                                                                                                       
New file to add: /games/Radon Break [01009160181BA000][US][v0].nsz

**Comment by acaltas** [comment]
or error on tinfoil itself

Unable to identify file v-teenage_mutant_ninja_turtles_shredders_revenge_v65536.nsp, app ID: None. The error was:

'NoneType' object has no attribute 'lower'

New file to add: /games/Teenage_Mutant_Ninja_Turtles/Teenage.Mutant.Ninja.Turtles.Shredders.Revenge.NSW/[0100FE701475A000] -teenage_mutant_ninja_turtles_shredders_revenge.nsp

New file to add: /games/The_Witcher_3_Wild_Hunt_Complete_Edition_NSW-VENOM/The Witcher [01003D100E9C6000].xci

Title ID not found in titledb: 01002DB007A96000

Title ID not found in titledb: 0100DCA0064A6000

Title ID not found in titledb: 01007EF00011E000

Title ID not found in titledb: 01006BB00C6F0000

Title ID not found in titledb: 010005E00E2BC000

Title ID not found in titledb: 01007900080B6000

Title ID not found in titledb: 0100D1B006744000

Title ID not found in titledb: 0100C0E00F0AC000

Title ID not found in titledb: 010065A014024000

Title ID not found in titledb: 0100C3C012718000

Title ID not found in titledb: 0100182014022000

Title ID not found in titledb: 01006D4003BCE000

Title ID not found in titledb: 0100D71004694000

Title ID not found in titledb: 0100D870045B6000

Title ID not found in titledb: 0100A5A00B2AA000

Title ID not found in titledb: 01001F201121E000

Title ID not found in titledb: 0100F4C009322000

Title ID not found in titledb: 01004E700DFE6000

Title ID not found in titledb: 01008D300C50C000

Title ID not found in titledb: 010036B0034E4000

Title ID not found in titledb: 0100EA80032EA000

Title ID not in versions.json: 010015100B514000

Title ID not found in titledb: 01003D100E9C6000

Scanning library...

**Reply by SiscuPrats** [reply]
No title id in name file Unable to identify file v-teenage_mutant_ninja_turtles_shredders_revenge_v65536.nsp,

you need to put the id in the file name between [] to identify it

**Reply by acaltas** [reply]
i now that some files are wrong but Title ID not found the main problem with a lot of Games witch use the correct format, as on the previous version it was working

**Reply by SiscuPrats** [reply]
It is still identifying my library. I will take some of the ids that you have not detected to see if I detect them as luigi's mansion for example.

**Reply by SiscuPrats** [reply]
More than 10 hours and it's still synchronizing... When I finish I will check if it detects those ids... but as I said before, if every time I have to reload the page it takes more than 10 hours, it is unfeasible.

**Reply by acaltas** [reply]
how many Titles you have to index looks like a lot if it takes 10h or more

**Reply by fpratsbu** [reply]
Arround 10k titles.

**Reply by acaltas** [reply]
yeah 10k titles are to much for this shop

**Reply by fpratsbu** [reply]
Normaly, the stores haves more games.

**Comment by Sev7nrayne** [comment]
@a1ex4 

Additional details after testing:
[Ownfoil gui vs game folder](https://github.com/a1ex4/ownfoil/issues/57)
